### DISCUSSION TITLE
<title>Authorized Github OAuth Apps

### DISCUSSION THREAD

**Discussion by ACreator-Global** [community/community#123317] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I could not get permission from Github  for  AWS Amplify, I am trying to deploy my app through Amplify, is there anyone know the right procedure to setup this connection between Github OAuth Apps and Amplify ? 
![auth_3](https://github.com/community/community/assets/168890627/ad0d1205-0e14-41b3-a73d-c2604b72d6d5)
![auth](https://github.com/community/community/assets/168890627/ee173d04-ba6e-4cc0-9635-a62270ffc7f0)
![auth_2](https://github.com/community/community/assets/168890627/1a1dc4d9-ce3f-4e1d-8681-eb5c70352a81)

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation, you need to verify it


Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!
